src/pages/PreReviewPage.tsx

```diff
@@ -17,11 +17,17 @@ const SummaryPage = () => {
       <Main>
         <MainChild center>
           <Breadcrumbs step={2} />
-          <Prose textCenter>
+          <Prose textCenter id="audiofocus">
             <h1 aria-label="Review Your Selections.">Review Your Selections</h1>
             <Text narrow>Confirm and change any votes as necessary.</Text>
+            <div aria-label="Use the down arrow, then" />
```

Comment:
Wondering about this like the previous one. I think we should only use full sentences/phrases to keep things as translatable as possible. And also avoid editing one part without the other.